lib/app/app_body.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:flutter/material.dart';
+import 'package:hw_2/main.dart';
+
+import 'package:hw_2/widgets/card_content.dart';
+import 'package:hw_2/widgets/carousel_indicator.dart';
+import 'package:hw_2/widgets/last_updated.dart';
+
+class AppBody extends StatelessWidget {
+  final Function() notifyParent;
+
+  const AppBody({Key? key, required this.notifyParent}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    var size = MediaQuery.of(context).size;
+    var orientation = MediaQuery.of(context).orientation;
+
+    var toolbarRowHeight =
+        orientation == Orientation.portrait ? size.height / 8 : size.width / 10;
+    var centerColumnWidth = size.width / 10;
+    var iconsSize = orientation == Orientation.portrait
+        ? toolbarRowHeight / 4
+        : toolbarRowHeight / 2;
+    var bodyTopPadding =

```

Comment:
use const/final where possible (so the reader understands that those are not variables but constant values)